### PR TITLE
chore: upgrade mocha, eslint, cross-env, nyc, webpack and whatwg-fetch

### PR DESCRIPTION
**Pull request by gchoqueux**
* Upgrade last packages in dev dependencies
* **Last current [node js 11.0.0](https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V11.md#2018-10-23-version-1100-current-jasnell) (2018-10-23) freeze test randomly.**
 I add **node 10 build** and remove **node 11 build** (11.0.0) for the moment.

### CODE REVIEW

**Comment by zarov** [comment]
Again ? What changed since the last upgrade ?

**Comment by gchoqueux** [comment]
All packages were not updated. And there is a problem with node js 11.0.0. (seePR message)

**Comment by gchoqueux** [comment]
@zarov PR is ready

File: test/examples/bootstrap.js [hunk 1]
```diff
@@ -133,6 +147,11 @@ before(async () => {
             if (container) {
                 container.style.display = 'none';
             }
+            const divScaleWidget = document.querySelectorAll('.divScaleWidget');
```
Comment (zarov) [inline test/examples/bootstrap.js:150]: Does it break something ?

Comment (gchoqueux) [reply test/examples/bootstrap.js:150]: Sometimes, this division interfere with mouse.down + mouse.move.

Comment (zarov) [reply test/examples/bootstrap.js:150]: Ok :+1:

**Review by zarov — APPROVED** [review]
Seems good to me, not a lot of change in fact, thanks for the hard work of upgrading all of these though !

**Pull request merged**